src/main/java/listfiles/ListFiles.java

```diff
@@ -0,0 +1,135 @@
+package listfiles;
+
+import java.io.*;
+import java.util.*;
+
+public class ListFiles {
+
+    static File inputFile;
+
+    private static String separator = System.getProperty("file.separator");
+
+    static int countLevel(String string, String delimiter) {
```

Comment:
Get rid of all static methods apart from main(). Create separate classes with non-static methods.